### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by mwmckee**

### CODE REVIEW

File: OneDriveConnector/OneDriveConnector.Tests/UnitTest.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+namespace OneDriveConnector.Tests
+{
+    [TestClass]
+    public class OneDriveConnectorTests
+    {
+        [TestMethod]
+        public void TestAccessTokenRequestFormatter()
+        {
+            // arange
```
Comment (apdutta) [inline OneDriveConnector/OneDriveConnector.Tests/UnitTest.cs]: Arrange typo

Comment (mwmckee) [inline OneDriveConnector/OneDriveConnector.Tests/UnitTest.cs]: fixed!

**Comment by apdutta** [comment]
Signoff

**Comment by JimGaleForce** [comment]
looks good

**Pull request merged**